scripts/build/build_boost.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Author : Alberto M. Esmoris Pena
+#
+# Build the boost library
+
+# ---  PREPARE ENVIRONMENT  --- #
+# ----------------------------- #
+# Load environment if it has not been loaded yet
+if [[ ${HELIOS_ENV} != 'ON' ]]; then
+    this_dir=$(dirname ${BASH_SOURCE[0]})
+    source ${this_dir}'/../local_env.sh'
+fi
+
+
+
+# ---  SCRIPT VARIABLES  --- #
+# -------------------------- #
+# Boost package
+BOOST_PKG='boost_1_76_0.tar.gz'
+# Boost URL
+BOOST_URL='https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/'
+BOOST_URL="${BOOST_URL}${BOOST_PKG}"
+# Boost download path
+BOOST_TAR="${HELIOS_LIB_DIR}${BOOST_PKG}"
+# Boost lib path (where it is extracted)
+BOOST_DEF_DIR="${HELIOS_LIB_DIR}"$(sed 's/\.tar\.gz//g' <<< ${BOOST_PKG})
+# Boost lib path (where it MUST be placed)
+BOOST_DIR="${HELIOS_LIB_DIR}"'boost/'
+# Target version that boost must link against
+BOOST_PYTHON_TARGET_VERSION='python3.7'
```

Comment:
We need to clarify that the python version must be edited accordingly to the one being used in the compilation.